packages/js/src/classic-editor/initial-state/index.js

```diff
@@ -0,0 +1,58 @@
+import {
+	getContent,
+	getDate,
+	getExcerpt,
+	getFeaturedImage,
+	getFocusKeyphrase,
+	getMetaDescription,
+	getPermalink,
+	getSeoTitle,
+	getSlug,
+	getTitle,
+	isCornerstone,
+} from "../dom";
+import { FOCUS_KEYPHRASE_ID } from "@yoast/seo-store";
+
+/**
+ * Gather the initial state of the classic editor.
+ *
+ * @returns {Object} The initial state of the classic editor.
+ */
+const getEditorInitialState = () => ( {
+	title: getTitle(),
+	date: getDate(),
+	permalink: getPermalink(),
+	excerpt: getExcerpt(),
+	content: getContent(),
+	featuredImage: getFeaturedImage(),
+} );
+
+/**
+ * Gathers the initial state of the metabox of the classic editor.
+ *
+ * @returns {Object} The initial state of the metabox of the classic editor.
+ */
+const getFormInitialState = () => ( {
+	seo: {
+		title: getSeoTitle(),
+		description: getMetaDescription(),
+		slug: getSlug(),
+		isCornerstone: isCornerstone(),
+	},
+	keyphrases: {
+		[ FOCUS_KEYPHRASE_ID ]: {
+			id: FOCUS_KEYPHRASE_ID,
+			keyphrase: getFocusKeyphrase(),
+		},
+	},
+} );
```

Comment:
When more initial state gets added to the form initial state this can get split up into different functions (one for `seo`, one for `keyphrases` etc.). I decided to leave it for now to not overcomplicate things.

Reply:
Very clean 👏